inst/package_report/package_vignette_template.Rmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "{{pkg_name}} Package Report, by pkgnet"
+output: 
+    rmarkdown::html_vignette:
+        toc: true
+vignette: >
+    %\VignetteIndexEntry{Package Report, by pkgnet}
+    %\VignetteEngine{knitr::rmarkdown}
+    %\VignetteEncoding{UTF-8}
+---
+
+This report on **{{pkg_name}}** is generated by [**pkgnet**](https://uptakeopensource.github.io/pkgnet/), an R package for analyzing other R packages through the lens of [network theory](https://en.wikipedia.org/wiki/Network_theory). 
+
+```{r setup, include=FALSE}
+library(pkgnet)
+
+knitr::opts_chunk$set(
+    echo = FALSE
+    , warning=FALSE
+    , out.width='100%'
+)
+pkgnet:::silence_logger()
+
+reporters <- {{pkg_reporters}}
+
+reporters <- lapply(
+          X = reporters
+          , FUN = function(reporter){
+              reporter$set_package(pkg_name = "{{pkg_name}}", {{pkg_path}} )
+              return(reporter)
+          }
+      )
+
+```
+
+```{r warning=FALSE}
+reportSections <- lapply(reporters, function(reporter) {
+  report_env <- list2env(list(reporter = reporter))
+  knitr::knit_child(
```

Comment:
this is nontrivial to do, cool that you figured it out. Every time I think I understand `knit_child()`, I hit some new confusing aspect of it

Reply:
This was actually a pretty minimal change from what is in the existing `package_report.Rmd`. I just deleted the bit about the tabs.